src/languages/es.js

```diff
@@ -422,6 +422,7 @@ export default {
         error: {
             firstNameLength: 'El nombre no debe tener más de 50 caracteres',
             lastNameLength: 'El apellido no debe tener más de 50 caracteres',
```

Comment:
Are we using these anymore?

Reply:
Yeah for the API error toasts that are still fired for the name calls